docs/learn/learn-nft-projects.md

```diff
@@ -88,127 +94,106 @@ KodaDot.
+- NFTs can be fractionalised (RFTs - refungible tokens)
+- Due to metadata format flexibility and mutability, it is possible and very simple to implement on-chain data obfuscation for enhanced privacy
+
+Unique Network provides a fully featured EVM with a few perks:
+
+- The EVM address space (0x…) is mapped to the corresponding Substrate address space which simplifies object and contract access between the EVM and Substrate objects via CrossAddressing mechanism. This means that you can mix and match technologies. For example you can have a native NFT call an EVM contract. This is the core mechanism for extending and customising NFTs for specific use cases.
+- This also means that a native Substrate object is addressable in EVM applications without an intermediary address handling contract, i.e. applications like the Metamask wallet can address Unique Network Substrate objects directly. This greatly improves efficiency.
+- Unique network EVM utilities contain interfaces to the blockchain’s core Substrate pallet functions which allows Solidity contracts to directly invoke them.
+- This interoperability is fully supported by the JS/TS SDK allowing rapid development even in the mix-and-match scenario.
+
+The recommended and most efficient way to build on Unique Network is via the JS/TS SDK version 2.0 at the moment of this writing. The SDK interacts with the blockchain via an RPC mechanism eliminating the need to build and maintain a complex development environment. It fully implements the [Unique Metadata Schema 2.0](https://docs.unique.network/reference/schemas/). The Unique SDK has recently incorporated the ability to create NFT assets on the Asset Hub as well which makes it a very versatile tool for NFT management and creation on the Polkadot ecosystem.
+
+Developer tools and documentation links:
 
-[Unique network](https://unique.network/), an NFT-specific blockchain offering innovations such as
-sponsored transactions, bundling fungible tokens with non-fungibles, and splitting NFTs into
-fungible tokens for partial ownership.
+- [General information](https://docs.unique.network/about)
+- [Substrate native advanced NFTs (NFT 2.0)](https://docs.unique.network/build/sdk/v2/quick-start.html) 
+- [Building NFTs in an EVM](https://docs.unique.network/build/evm/)
+- [Tutorials/Cookbook](https://docs.unique.network/tutorials)  
+- [Youtube Workshops and Tutorials](https://www.youtube.com/watch?v=StfRap-dvks&list=PLFSU2-RmX858vLu9a5bukOTpJ1dYzsE4o)
+- [Reference Documentation](https://docs.unique.network/reference)
+- [NFT Marketplace](https://unqnft.io/market?filterState=)
+- [Indexer/Block Explorer](https://uniquescan.io/unique/)
 
-Unique Network have launched two NFT projects to date: Substrapunks as part of
-[Hackusama](https://hackusama.devpost.com/), and Chelobricks as a promotion during
-[Polkadot Decoded](https://decoded.polkadot.network/).
+The most prominent projects that currently implement Unique Network’s advanced NFT capabilities are
```

Comment:
What about Sovereign Nature?

Reply:
The project has officially ceased operation.